feathr_project/feathr/registry/registry_utils.py

```diff
@@ -87,13 +87,17 @@ def transformation_to_def(v: Transformation) -> dict:
     elif isinstance(v, WindowAggTransformation):
         ret = {
             "defExpr": v.def_expr,
+            "def_expr": v.def_expr,
```

Comment:
Is this duplicated? I remember this is already in the code?

Reply:
This one is for fixing old purview client.  the similar code that is already in the code is for registry api